python/acquire/__init__.py

```diff
@@ -204,7 +196,23 @@ def is_not_done() -> bool:
         runtime.stop()
         logging.info("STOPPED")
 
+    viewer.layers.clear()
     do_acquisition()
 
 
+_g_runtime: Optional[Runtime] = None
+"""The global acquire runtime."""
+
+
+def _get_runtime() -> Runtime:
+    """Potentially create and get the global acquire runtime."""
+    global _g_runtime
+    if _g_runtime is None:
+        logging.info("INITING RUNTIME")
+        _g_runtime = acquire.Runtime()
+    else:
+        logging.info("REUSING RUNTIME")
+    return _g_runtime
+
+
 # TODO: (nclack) add context manager around runtime and start/stop
```

Comment:
I was tempted to tackle this TODO too, but I guess this needs to be done in `runtime.rs`?